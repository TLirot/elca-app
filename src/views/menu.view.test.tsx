import { shallow, ShallowWrapper } from "enzyme";
import { MenuView } from "./menu.view";

const initShallowComponent = (): ShallowWrapper<Record<never, never>, Record<never, never>, MenuView> => {
    return shallow(<MenuView/>);
}

describe("MenuView Test =>", () => {
    it("Should render correctly", () => {
        expect(initShallowComponent()).toMatchSnapshot();
    });
});
