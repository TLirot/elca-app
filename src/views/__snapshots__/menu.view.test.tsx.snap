// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuView Test => Should render correctly 1`] = `
<Nav
  fill={false}
  justify={false}
>
  <Link
    className="nav-link"
    to="/bitcoin"
  >
    Bitcoin
  </Link>
  <Link
    className="nav-link"
    to="/fibonacci"
  >
    Fibonacci
  </Link>
</Nav>
`;
